bin/build-helper

```diff
@@ -11,14 +11,16 @@ if [[ -n "$REGISTRY" ]]; then  # if non-empty
   REGISTRY="$(echo $REGISTRY | sed 's#/$##')/"  # ensure '/' suffix
 fi
 
+: ${ENV:="dev"}  # assign default if not passed in
+
```

Comment:
The established convention is to use `NODE_ENV=development|production` for build tooling like this. Is there a reason to deviate in this case?